src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3827,6 +3829,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <ResolveManifestFiles
         TargetFrameworkVersion="$(TargetFrameworkVersion)"
+        TargetFrameworkIdentifier="$(TargetFrameworkIdentifier)"
```

Comment:
@BenVillalobos fyi this will manifest as a break if they have Tasks GACed. So we totally need your work :)